### PR TITLE
Bump phpstan level

### PR DESCRIPTION
**Pull request by VincentLanglet**
## Subject

I am targeting this branch, because {reason}.

## Changelog

```markdown
### Fixed
- Some phpdoc
```

### CODE REVIEW

File: src/Block/BlockServiceManager.php [hunk 1]
```diff
@@ -22,7 +22,7 @@
 final class BlockServiceManager implements BlockServiceManagerInterface
 {
     /**
-     * @var array
+     * @var array<string, string|BlockServiceInterface>
```
Comment (core23) [inline src/Block/BlockServiceManager.php:25]: This is correct?

Comment (VincentLanglet) [reply src/Block/BlockServiceManager.php:25]: it was to me.

If I understood correctly it was either a service or the id of the service and when trying to accessing it, it was loaded.

**Comment by jordisala1991** [comment]
@VincentLanglet do you have time to finish this one?

**Comment by VincentLanglet** [comment]
Sure

**Comment by VincentLanglet** [comment]
Done @jordisala1991

File: composer.json [hunk 2]
```diff
@@ -80,5 +84,13 @@
...
             "Sonata\\BlockBundle\\Tests\\": "tests/"
         }
+    },
+    "scripts": {
+        "post-install-cmd": [
+            "[ $COMPOSER_DEV_MODE -eq 0 ] || vendor/bin/simple-phpunit install"
+        ],
+        "post-update-cmd": [
+            "[ $COMPOSER_DEV_MODE -eq 0 ] || vendor/bin/simple-phpunit install"
+        ]
```
Comment (jordisala1991) [inline composer.json]: this is not needed anymore.

Comment (VincentLanglet) [reply composer.json]: done

**Review by jordisala1991 — APPROVED** [review]

**Review by core23 — APPROVED** [review]

**Comment by core23** [comment]
We should avoid such big PRs and create smaller ones. They are hard to review and will create big feedback loops after fixing the comments.

**Pull request merged**